### PR TITLE
Readme: Add blurred transparent bg for GNOME shell

### PR DESCRIPTION
**Pull request by Marenz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ for everyday use. It is aiming for power users with a modern feature mindset.
 - ✅ High-DPI support.
 - ✅ Vertical Line Markers (quickly jump to markers in your history!)
 - ✅ Vi-like input modes for improved selection and copy'n'paste experience and Vi-like `scrolloff` feature.
-- ✅ Blurred behind transparent background when using Windows 10 or KDE window manager on Linux.
+- ✅ Blurred behind transparent background support for Windows 10, the KDE window manager and the GNOME shell on Linux.
```
Comment (Yaraslaut) [inline README.md:25]: I assume that one Windows 11 it also works, and we can write that we support both KDE and GNOME desktop environments

Comment (Marenz) [reply README.md:25]: @christianparpart is windows 11 also supported? I could write Windows 10+ in that case

Comment (christianparpart) [reply README.md:25]: Yes, Windows 10 and above. :)

**Comment by Marenz** [comment]
Updated

File: README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ for everyday use. It is aiming for power users with a modern feature mindset.
 - ✅ High-DPI support.
 - ✅ Vertical Line Markers (quickly jump to markers in your history!)
 - ✅ Vi-like input modes for improved selection and copy'n'paste experience and Vi-like `scrolloff` feature.
-- ✅ Blurred behind transparent background when using Windows 10 or KDE window manager on Linux.
+- ✅ Blurred behind transparent background support for Windows 10 and above as well as the KDE and GNOME desktop environment on Linux.
```
Comment (cqexbesd) [inline README.md:25]: What is "Blurred behind transparent background support"? Is it meant to be "Transparent and translucent (blurred) background support"?

Also presumably it is for KDE and GNOME on all platforms? Or is it really Linux specific?

Comment (Yaraslaut) [reply README.md:25]: Users have the ability to control the transparency of their windows through a compositor, such as Picom or the standard ones for KDE and GNOME. 
This feature adds a blur effect to the background image visible when looking through a transparent window, creating following:
Image from the web:
image

Comment (christianparpart) [reply README.md:25]: Looks like we want to add a little video demo to https://contour-terminal.org/features/ as a sub-page, to illustrate this, maybe? 🤔

Comment (cqexbesd) [reply README.md:25]: Transparent and blurred backgrounds when run under Windows 10 (or above), GNOME or KDE, or with other compatible compositors.

?

**Review by christianparpart — APPROVED** [review]
many thanks for the active participation. I think it's good as it is proposed.

Generally, I think, it came quite positively out, that it might make sense to create a very little video (/photo?) to better illustrate what was meant, in the official docs.

**Pull request merged**